### PR TITLE
Add alternative source for EDGT Lyon

### PR DESCRIPTION
**Pull request by Nitnelav**
There is a way to get Lyon EDGT from the ADISP portal : http://www.progedo-adisp.fr/serie_emd.php
It is free for academic use and should be the most straightforward way to get the data.
The data is formatted a bit differently compared to what @sebhoerl got (driectly from CEREMA), so this PR adds the code to read EDGT from this new source.

### CODE REVIEW

File: docs/cases/lyon.md [hunk 1]
```diff
@@ -51,19 +51,40 @@ Download all the *zip*'d GTFS schedules and put them into the folder `data/gtfs`
...
+- `data/edgt_lyon_2015/lyon_2015_std_tel_men.csv`
+- `data/edgt_lyon_2015/lyon_2015_std_faf_pers.csv`
+- `data/edgt_lyon_2015/lyon_2015_std_tel_pers.csv`
+- `data/edgt_lyon_2015/lyon_2015_std_faf_traj.csv`
+- `data/edgt_lyon_2015/lyon_2015_std_tel_traj.csv`
+- `data/edgt_lyon_2015/EDGT_AML2015_ZF_GT.DAT`
+- `data/edgt_lyon_2015/EDGT_AML2015_ZF_GT.ID`
+- `data/edgt_lyon_2015/EDGT_AML2015_ZF_GT.IND`
+- `data/edgt_lyon_2015/EDGT_AML2015_ZF_GT.MAP`
+- `data/edgt_lyon_2015/EDGT_AML2015_ZF_GT.TA`
```
Comment (sebhoerl) [inline docs/cases/lyon.md:87]: TA or TAB ?

File: docs/cases/lyon.md [hunk 2]
```diff
@@ -126,6 +147,7 @@ is used, choose the updated HTS in the config:
 config:
   # ...
   hts: edgt_lyon
+  edgt_lyon_source: adisp  # adisp/cerema
```
Comment (sebhoerl) [inline docs/cases/lyon.md:150]: Can you add a comment somewhere in the text above to make sure it is clear that one needs to make a choice here?

File: data/hts/edgt_lyon/filtered.py [hunk 3]
```diff
@@ -6,7 +6,15 @@
 """
 
 def configure(context):
-    context.stage("data.hts.edgt_lyon.cleaned")
+    edgt_lyon_source = context.config("edgt_lyon_source", "adisp")
```
Comment (sebhoerl) [inline data/hts/edgt_lyon/filtered.py:9]: Instead of "adisp", can we have a default value "unchosen" ...

File: data/hts/edgt_lyon/filtered.py [hunk 4]
```diff
@@ -6,7 +6,15 @@
...
+
+    if edgt_lyon_source == "adisp":
+        context.stage("data.hts.edgt_lyon.cleaned_adisp", alias="data.hts.edgt_lyon.cleaned")
+    elif edgt_lyon_source == "cerema":
+        context.stage("data.hts.edgt_lyon.cleaned_cerema", alias="data.hts.edgt_lyon.cleaned")
+    else:
```
Comment (sebhoerl) [inline data/hts/edgt_lyon/filtered.py:15]: And then here another elif that raises an exception, but explicitly telling the user that the EDGT source needs to be chosen, if the value is still "unchosen". Then we can nicely remind the user to set it :)

**Review by sebhoerl — CHANGES_REQUESTED** [review]
Really nice, could you please look at the few little points?

File: docs/cases/lyon.md (hunk 1 above)
Comment (Nitnelav) [reply docs/cases/lyon.md]: TAB ;)

**Comment by Nitnelav** [comment]
I added a small .gitignore update there too, unrelated to the PR but not worth a seperate one either i rekon

**Review by sebhoerl — APPROVED** [review]

**Pull request merged**